app/roles/virtual-machine/tasks/host_drives.yml

```diff
@@ -0,0 +1,26 @@
+- name: inspect vm for desired drive mapping
+  shell: virsh dumpxml --inactive {{ virtual_machine_name }} | grep "<source dev='/dev/{{ drive }}'[ '=0-9a-z]*/>"
```

Comment:
why use a regex here to match the drive?

Reply:
the source block will sometimes have additional attributes that are set and manged by libvirt.